CONTRIBUTING.md

```diff
@@ -105,7 +105,12 @@ We don't follow any other convention, but if you want to use one, we recommend [
 
 ### Changesets <!-- omit in TOC -->
 
-We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to add a changeset by running `yarn changeset` which contains what packages should be bumped, their associated semver bump types and some markdown which will be inserted into changelogs.
+We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to [add a changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) by running `yarn changeset` which contains what packages should be bumped, their associated semver bump types and some markdown which will be inserted into changelogs.
+See [documentation](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) on how to add a changeset.
+
+These changesets are added in the `.changesets` directory at the root of the repository, that are later removed [on release](#release-on-github-and-npm). These generated changesets files should be commited to your PR.
```

Comment:
Do you think we should specify that the changesets are removed automatically? :)
Plus, the order of these 2 sentences is weird. First, we speak about their removal, and then we say that we should commit them :D  I suggest something like this:
```
These changesets are added in the `.changesets` directory at the root of the repository, and should be commited to your PR.
Once you made a release, these files are automatically removed.
```

Reply:
Simplified it with your suggestions: 

These changesets are added in the `.changesets` directory at the root of the repository, and should be added in your PR.
Once there is a [release](#release-on-github-and-npm), these files are automatically removed.